core/src/test/java/com/linecorp/armeria/common/HttpResponseTest.java

```diff
@@ -90,4 +90,17 @@ void statusOfResponseHeadersShouldNotBeInformational() {
                                                  HttpData.ofUtf8("bob")))
                 .isInstanceOf(IllegalArgumentException.class).hasMessageContaining("non-1xx");
     }
+
+    @Test
+    void ofRedirect() {
+        final HttpResponse res = HttpResponse.ofRedirect("locationFor");
+        final AggregatedHttpResponse aggregatedRes = res.aggregate().join();
+        assertThat(aggregatedRes.status()).isEqualTo(HttpStatus.TEMPORARY_REDIRECT);
```

Comment:
Could add an assertion for headers as well

Reply:
Did you mean check `location` header?
I added this like below.
```
assertThat(aggregatedRes.headers().get(HttpHeaderNames.LOCATION)).isEqualTo("locationFor");
```